README.md

```diff
@@ -53,7 +53,7 @@ final steps to make use of it in your project:
 
 ```javascript
 // Assuming you use use ES6 imports...
-import * as NineSlice from 'phaser3-nineslice'
+import {Plugin as NineSlicePlugin} from 'phaser3-nineslice'
```

Comment:
Is this just a stylistic difference or is there a deeper cause behind changing the import?

Reply:
Mostly style, but also because I like to import only the thing I need.

The other part would be that in a readme it gives the reader the idea that there is more being exposed than just a Plugin class when importing the package